examples/preprocessing/plot_scaling_importance.py

```diff
@@ -137,3 +180,42 @@
 plt.tight_layout()
 
 plt.show()
+
```

Comment:
I think that we need to describe the plot here (a better separability for instance).